test/dst.test.ts

```diff
@@ -104,7 +106,7 @@ async function main() {
       const testRandomSeed = Math.random();
       console.log("testRandomSeed: " + testRandomSeed);
       const { context: currentContext, promise: currentPromise } = resonate._run("test", "baseline", {
-        testFailureProb: 0.6,
+        testFailureProb: probFailure,
```

Comment:
Is it okay if we just call this opt key `test` and remove the `testRandomSeed`?

Reply:
opt key renamed to test.